core/lib/pattern_assembler.js

```diff
@@ -45,6 +45,7 @@ var pattern_assembler = function () {
         return patternlab.patterns[i];
       }
     }
+    console.trace("Here I am!");
```

Comment:
curious as to why i dont see this message show up in my output. remove when you no longer need this